frontend-maven-plugin/src/it/yarn-integration/verify.groovy

```diff
@@ -0,0 +1,8 @@
+assert new File(basedir, 'target/.buildenv/node').exists() : "Node was not installed in the custom install directory";
+assert new File(basedir, 'node_modules').exists() : "Node modules were not installed in the base directory";
+
+import org.codehaus.plexus.util.FileUtils;
+
+String buildLog = FileUtils.fileRead(new File(basedir, 'build.log'));
+
+assert buildLog.contains('BUILD SUCCESS') : 'build was not successful'
```

Comment:
Maybe also assert that node_modules/less/package.json exists? Then we're sure the plugin has actually installed dependencies, not just short-circuited!